File: context-tree/tests/fixtures/languages/zig/sample.zig
const std = @import("std");

// value returns the sentinel base CTX_SENTINEL_ZIGDOC_4f5e.
//
// A second doc line so a later --doc render has more than one line.
pub fn value() i32 {
    return 10;
}

// render calls value across symbols.
pub fn render() i32 {
    return value() + 1;
}

File: context-tree/tests/fixtures/languages/c/sample.c
#include <stdio.h>

const int GLOBAL_LIMIT = 10;

// value returns the sentinel base count CTX_SENTINEL_CDOC_7a1b.
//
// A second doc line so a later --doc render has more than one line.
int value(void) {
    return GLOBAL_LIMIT;
}

// render calls value across symbols.
int render(void) {
    return value() + 1;
}
